README.md

```diff
@@ -55,7 +55,32 @@ _Note:_ If you run Flood for Transmission behind SSL and in Chrome you can also
 
 ### [BETA] Customization
 
-1. You can configure the common download path by setting the appropriate value in the pre-config file: `flood-for-transmission/public/config.json`. You can open it with your favourite editor or write config with the next command: `echo '{"COMMON_PATH":"/downloads/expeliarmus,/downloads/lumos"}' > flood-for-transmission/public/config.json`
+1. All User Interface defaults can be customized in `/flood-for-transmission/public/config.json` and should be self-explanatory, `cp config.json.defaults config.json` to get started:
+* `"DARK_MODE"`
+  * Type: String e.g. `"auto"` or `"enabled"` or `"disabled"`
+  * Default: `"auto"`
+* `"SWITCH_COLORS"`
+  * Type: Boolean e.g `true` or `false`
+  * Default: `false`
+* `"NOTATION_24H"`
+  * Type: Boolean e.g `true` or `false`
+  * Default: `true`
+* `"WRAP_HEADER"`
+  * Type: Boolean e.g. `true` or `false`
+  * Default: `false`
+* `"COMMON_PATH"`
+  * Type: Array of Strings e.g. `["/downloads/expeliarmus","/downloads/lumos"]`
```

Comment:
🧙

Reply:
No idea what this (:mage: :eyes:) means???

Reply:
Sorry, I was making a joke, referring to the Harry Potter spells that you're using here, but I understand that wasn't entirely clear. ;)